asset/cloud-client/quickstart_exportassets_test.py

```diff
@@ -43,8 +44,8 @@ def asset_bucket(storage_client):
 
 
 def test_export_assets(asset_bucket, capsys):
-    dump_file_path = "gs://", asset_bucket, "/assets-dump.txt"
```

Comment:
It seems strange that you're passing on line 40 and silencing the exception -- are you regularly seeing exceptions when cleaning up?